### ISSUE TITLE
Discrepancy in number of pairs in training set

### ISSUE THREAD

**Issue by chinnusai25** [athn-nik/teach#10]
Paper states that, 
` There are approximately 5.7k and 23.4k pairs in the validation and training sets respectively. `
But when training is executed we get as below,
Loading BABEL train: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 6601/6601 [00:53<00:00, 124.08it/s]
[teach.data.babel][INFO] - Processed 6601 sequences and found 3091 invalid cases based on the datatype.
[teach.data.babel][INFO] - **15863 sequences -- datatype:separate_pairs**.
[teach.data.babel][INFO] - 14.13% of the sequences which are rejected by the sampler in total.
[teach.data.babel][INFO] - 0.0% of the sequence which are rejected by the sampler, because of the excluded actions.
[teach.data.babel][INFO] - 14.13% of the sequence which are rejected by the sampler, because they are too short(<0.5 secs) or too long(>25.0 secs).
[teach.data.babel][INFO] - Discard from BML: 0
[teach.data.babel][INFO] - Discard not KIT: 0
Loading BABEL val: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2189/2189 [00:17<00:00, 124.54it/s]
[teach.data.babel][INFO] - Processed 2189 sequences and found 983 invalid cases based on the datatype.
[teach.data.babel][INFO] - 5672 sequences -- datatype:separate_pairs.
[teach.data.babel][INFO] - 16.27% of the sequences which are rejected by the sampler in total.
[teach.data.babel][INFO] - 0.0% of the sequence which are rejected by the sampler, because of the excluded actions.
[teach.data.babel][INFO] - 16.27% of the sequence which are rejected by the sampler, because they are too short(<0.5 secs) or too long(>25.0 secs).
[teach.data.babel][INFO] - Discard from BML: 0
[teach.data.babel][INFO] - Discard not KIT: 0

which results that the number of training pairs are 15.8k which doesn't match with that of in paper (i.e 23.4k)

**Comment by atnikos** [comment]
Indeed there are around 5.7k pairs for the validation set. I have to check the training stats you are probably right. If you lower the rejection threshold of the sampler(min_len to 10) I think you will get the amount I am referring to. Be aware, the extracted data are different from the sequences. the sequences are AMASS sequences and we extract the data from those sequences.
